src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -11,34 +11,29 @@
 
 import java.util.Map;
 
-@SuppressWarnings ({"squid:S1854", "squid:S2160"})  // suppress unneeded int *groupOrder variables, equals method
+@SuppressWarnings({"squid:S1854", "squid:S2160"})  // suppress unneeded int *groupOrder variables, equals method
 public class CosmosDBConfig extends AbstractConfig {
-    private static final Validator NON_EMPTY_STRING = new NonEmptyString();
-    
     public static final String COSMOS_CONN_ENDPOINT_CONF = "connect.cosmos.connection.endpoint";
+    public static final String COSMOS_CONN_KEY_CONF = "connect.cosmos.master.key";
+    public static final String COSMOS_DATABASE_NAME_CONF = "connect.cosmos.databasename";
+    public static final String COSMOS_CONTAINER_TOPIC_MAP_CONF = "connect.cosmos.containers.topicmap";
+    public static final String COSMOS_PROVIDER_NAME_CONF = "connect.cosmos.provider.name";
+    public static final int COSMOS_DATABASE_GROUP_ORDER = 2;
+    public static final String COSMOS_CLIENT_USER_AGENT_SUFFIX = "APN/1.0 Microsoft/1.0 KafkaConnect/";
+    public static final String TOLERANCE_ON_ERROR_CONFIG = "tolerance.error";
+    public static final String TOLERANCE_ON_ERROR_DOC = "Tolerance level. None for failing on error. All for log and continue";
```

Comment:
is this TOLERANCE_ON_ERROR_DOC correct? 

it says None for failing on error, should it say "error" as the config line above it seems to be setting TOLERANCE_ON_ERROR_CONFIG = "tolerance.error". 

should it say "tolerance.all for log & continue" ? 

If i set TOLERANCE_ON_ERROR_CONFIG = "All" as this documentation says, what will happen?

Reply:
Do we even need a new config here? 
Kafka Connect has ```errors.tolerance``` which can be set to 'all', 'none'
That seems to do the same thing?